### ISSUE TITLE
Encoding a physical description outside the FRBR module not possible

### ISSUE THREAD

**Issue by lpugin** [music-encoding/music-encoding#1078]
Various refactoring steps since MEI 3.0 have made it not possible to use `<physDesc>` outside the FRBR module. This is problematic for cases where the metadata is about a single source and where providing some metadata about the physical description is desirable. In MEI 3.0, it was possible to do
```xml
<source>
   <titleStmt>
      <title>Soprani</title>
   </titleStmt>
   <physDesc>
      <!-- ... -->
   </physDesc>
</source>
```
and I think this is still desirable. 

Furthermore, `<physDesc>` is part of the `header` module, and it seems awkward that it cannot be used without `FRBR`. See a related discussion [here](https://github.com/music-encoding/music-encoding/issues/803#issuecomment-850473806)

I would suggest to investigate a possible fix by allowing (back) `<physDesc>` within `<source>` but I have not evaluated the consequences of this. Any thoughts?

**Comment by musicEnfanthen** [comment]
Pinging @music-encoding/ig-metadata

**Comment by doerners** [comment]
Giving my thought with the disclaimer that they might not be fully developed:

If any encoder wants to denote that only one single source exists that accounts e.g. for something encoded in the body they could describe this with the FRBR module neatly using `<manifestation singleton="true">`. So something like:

```XML
   <manifestation singleton="true">
      <titleStmt>
         <title>Soprani</title>
      </titleStmt>
      <physDesc>
         <!-- ... -->
      </physDesc>
   </manifestation>
```

If the encoder does'nt want to use the FRBR module (for whatever reason this might be) this could have a negative effect on data interoperability. Especially since the introduction of RDA my perception was that the FRBR concepts are essential in cataloguing practice (at least for larger GLAM institutions) and it would probably be very hard to exchange data with e.g. libraries.

Furthermore the definition of `<source>` states that it holds: 

> A bibliographic description of a source used in the creation of the electronic file.

My perception was that `<source>` points to secondary literature rather than e.g. a manuscript (maybe I'm mistaken here). From this understanding/point of view it doesn't seem too sensible to have `<physDesc>` within `<source>`. 

But maybe there is something I'm missing or simply misunderstand? So I guess my question would be why one wouldn't want to use the FRBR module?

**Comment by ahankinson** [comment]
The `header` and `frbr` modules in MEI should be independent, but the issue is that there is currently no way to use parts of `header` without `frbr`. `<physDesc>` can't be used without going through `<item>`, as it is the only valid parent element.

So I think the intent of this change is to simply make them independent again. 

Good point about `<source>` being for secondary literature... We'll have to check that out.

---

As for why: I have my own, personal, and largely negative views of FRBR. I think we should still provide FRBR functionality within MEI, but I also think there should be alternatives for those who do not want to do so. There will be no changes to the FRBR output so those who wish to do so can still do it, but fixing this will also let those who do not want to use FRBR to do that too.

(I could write several pages of why I think FRBR is very bad, but I will save you all from this... Let's just say I used to be a fan, but now think it's actively malicious for most cataloguing purposes.)

**Comment by riedde** [comment]
Yes, some header element are only available in elements from the frbr module. I also think that it should be possible to make them more independent. We'll have to evaluate on this, because I think there is more to do that just add physDesc to another element. Thanks for bringing this up!

**Comment by ahankinson** [comment]
@doerners I had a look at `<source>`

> My perception was that <source> points to secondary literature rather than e.g. a manuscript (maybe I'm mistaken here). 

In this instance I think that the "bibliographic description" is referring to a description of the original source that was used to create an electronic copy of the notation. So it would be acceptable to reference the book or PDF or whatever (with all its bibliographic and/or physical descriptions) that a particular musical piece was taken from.

This is how it is used in some of the [Sample Encodings](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Aguado_Walzer_G-major.mei#L68-L97).

**Comment by lpugin** [comment]
> If the encoder does'nt want to use the FRBR module (for whatever reason this might be) this could have a negative effect on data interoperability. Especially since the introduction of RDA my perception was that the FRBR concepts are essential in cataloguing practice (at least for larger GLAM institutions) and it would probably be very hard to exchange data with e.g. libraries.

Possibly. One could also argue that not using FRBR where this is not clearly necessary of where the model is not really appropriate - by being maybve either too simple or too complicated – then you would increase interoperability with other encodings, such as [TEI](https://tei-c.org/release/doc/tei-p5-doc/en/html/HD.html#HD3) for example. It would not be a one to one mapping but it would have a much more similar structure. There are many cases where, for good reasons, MEI follows a similar but not identical approach to TEI, and that would be another one.

**Comment by doerners** [comment]
Thanks @ahankinson and @lpugin for the helpful comments and of course for bringing this up. @riedde and me had a quick chat about this issue yesterday and our conclusion was that this seems to be a larger/more complicated issue we should discuss further. We will bring it up during the next ig meeting and if you (@ahankinson, @lpugin) find the time to join us we think this could really help the discussion. The meeting will take place on Wed, 1.3., 15h (UTC +1).

**Comment by pe-ro** [comment]
I'd like to propose adding a new member to the biblLike class, the members of which are allowed in `<source>`.  The`<biblFull>` element's content model will allow a mixture of data hitherto allowed only by the FRBR module.  That is, it will contain a mixture of work-, expression-, and manifestation-level info.

This addition will fix this issue.  Also, since `<biblFull>` will allow physical description of a manifestation, #1122 can be closed as well.

PR to follow soon.

**Comment by bwbohl** [comment]
@riedde and I merged the corresponding pull requests into the branch develop-profileDescPart, which is the head of the Pull Request #1367 

Please review it if it now fulfils all the intentions and feel free to approve ;-)

**Comment by pe-ro** [comment]
I'd very much like to put this revision to bed, but the revision and discussion are so convoluted it's difficult for me to sign off.  Can someone generate a schema with this revision included so that we can test its suitability?

**Comment by riedde** [comment]
@pe-ro here is a compiled version of mei-all based on #1367 hash 5a7fab8

[mei-all_pull1367_5a7fab8.rng.zip](https://github.com/user-attachments/files/23964761/mei-all_pull1367_5a7fab8.rng.zip)

**Comment by rettinghaus** [comment]
You can find the corresponding built schema directly in the PR, see: https://github.com/music-encoding/music-encoding/actions/runs/19766002751

**Comment by pe-ro** [comment]
Thanks, @riedde and @rettinghaus.

**Comment by riedde** [comment]
fixed in #1367
